model/registry.go

```diff
@@ -107,8 +107,8 @@ type IstioRegistry struct {
 	ConfigRegistry
 }
 
-// RouteRules lists all rules in a namespace (or all rules if namespace is "")
-func (i *IstioRegistry) RouteRules(namespace string) []*proxyconfig.RouteRule {
+// ListRouteRules lists all rules in a namespace (or all rules if namespace is "")
+func (i *IstioRegistry) ListRouteRules(namespace string) []*proxyconfig.RouteRule {
 	out := make([]*proxyconfig.RouteRule, 0)
 	rs, err := i.List(RouteRule, namespace)
```

Comment:
For future PRs, can we keep changes well scoped to the intent of the PR? Or at the very least, split into multiple commits, such that a commit that does renaming functions, etc., is on its own, while a commit that changes functionality is another. [Standard practice in kernel community]. Makes it easier to bisect issues and revert the offending commit.